_posts/2021-06-10-solidity-0.8.5-release-announcement.md

```diff
@@ -0,0 +1,162 @@
+---
+layout: post
+published: true
+title: 'Solidity 0.8.5 Release Announcement'
+date: '2021-06-10'
+author: Solidity Team
+category: Releases
+---
+
+[Solidity v0.8.5](https://github.com/ethereum/solidity/releases/tag/v0.8.5) allows conversions
+from ``bytes`` to ``bytesNN`` values, adds the ``verbatim``builtin function to inject
+arbitrary bytecode in Yul and fixes several smaller bugs.
+
+
+## Notable New Features
+### Bytes Conversion
+
+This release introduces the ability to convert ``bytes`` and ``bytes`` slices to fixed bytes types ``bytes1`` / ... / ``bytes32``.
```

Comment:
Maybe add: "While conversion between fixed-length bytes types has always been possible, it is now also possible to convert dynamically-sized `bytes` types to fixed-length bytes types.